src/components/AppLayout/Sidebar/SafeHeader/index.tsx

```diff
@@ -158,7 +164,7 @@ const SafeHeader = ({
           <ButtonHelper onClick={onReceiveClick}>
             <Icon size="sm" type="qrCode" tooltip="Show QR" />
           </ButtonHelper>
-          <CopyToClipboardBtn textToCopy={address} />
+          <CopyToClipboardBtn textToCopy={copyChainPrefix ? `${shortName}:${address}` : `${address}`} />
```

Comment:
CopyToClipboardBtn is used in two other places, should we also create a wrapper for it?

Reply:
I was implementing a wrapper for it called `CopyAddressToClipboardBtn` but the `CopyToClipboardBtn` is only copying an address in the `SafeHeader` component. 
<img width="198" alt="Screen Shot 2021-11-01 at 11 16 34" src="https://user-images.githubusercontent.com/32431609/139656782-1c9bafd1-7501-49e3-bccf-d0887e7af285.png">
Is it worthy to implement the wrapper afterall?